Makefile

```
@@ -147,13 +165,37 @@ logs: ## display all services logs (follow mode)
 	@$(COMPOSE) logs -f
 .PHONY: logs
 
-start: ## start all development services
-	@$(COMPOSE) up --force-recreate --build -d frontend-dev backend-dev worker-dev mta-in --wait
+start-deps: ## start the slow infra deps (postgres, redis, keycloak) in the background so they warm up while the rest of bootstrap runs
+	@$(COMPOSE) up -d --no-recreate postgresql redis keycloak
+.PHONY: start-deps
+
+start: ## start the light dev stack (backend, worker, frontend, keycloak, postgresql, redis)
```

checkmake (0.3.2)

[warning] Target body for "start" exceeds allowed length of 5 lines (8).

(maxbodylength)